package.json

```diff
@@ -70,6 +70,7 @@
     "htmlparser2": "^7.2.0",
     "localforage": "^1.10.0",
     "lodash": "4.17.21",
+    "memoize-one": "^6.0.0",
```

Comment:
Out of curiosity, why did we choose to add a new lib instead of using `_.memoize`: https://underscorejs.org/#memoize ?

Reply:
`_.memoize` didn't work for me out of the box because it creates a hash from the first argument of the function (vs checking to see if all arguments are unique). `memoize-one` "just worked" so I rolled with it instead of trying to figure out the hash function stuff.